client/app/types/course/statistics/assessmentStatistics.ts

```diff
@@ -12,6 +12,7 @@ interface AssessmentInfo {
 interface MainAssessmentInfo extends AssessmentInfo {
   isAutograded: boolean;
   questionCount: number;
+  isLiveFeedbackEnabled: boolean;
```

Comment:
why is it still `isLiveFeedbackEnabled` here while the associated jbuilder for this one is already changed to `liveFeedbackEnabled`?

In light of this, please also carefully check if all the related `isLiveFeedbackEnabled` is changed accordingly

Reply:
I think you might have been looking at a old version, it has already been changed